packages/restlessness-core/src/JsonScheduleEvents/index.ts

```diff
@@ -0,0 +1,88 @@
+import JsonConfigFile, { JsonConfigEntry } from '../JsonConfigFile';
+import PathResolver from '../PathResolver';
+import JsonServices from '../JsonServices';
+import Misc from '../Misc';
+import fsSync, { promises as fs } from 'fs';
+import path from 'path';
+import { exporterTemplate, indexTemplate } from './templates';
+import { promisify } from 'util';
+import rimraf from 'rimraf';
+
+export interface JsonScheduleEventsEntry extends JsonConfigEntry {
+  name: string
+  description?: string
+  rate: string
+  enabled?: boolean
+  serviceName: string
+  input?: {
+    type: 'input' | 'inputPath' | 'inputTransformer'
+    value: { [key: string]: any }
+  }
+  safeFunctionName: string
+}
```

Comment:
I think it should have daos array as well

Reply:
We should modify also the LambdaHandler (currently it handles only endpoints) or define another handler expecially for schedule events. This way daos packages hooks will be executed

Reply:
Correct, I was thinking something about a ScheduleHandler or CronHandler :-)

Reply:
Ok ScheduleHandler 👍